### ISSUE TITLE
API was shutdown

### ISSUE THREAD

**Issue by da2x** [BATgrowth/detect-bat-publishers#7]
https://mercury-proxy.privateinternetaccess.com/v3/publisher/identity?publisher=example.com is dead.

**Comment by da2x** [comment]
I’m not sure where they’re downloading it from, but the new Brave includes every publisher domain in a file in the user profile called publisher_list.

**Comment by maxenceC** [comment]
Hello @da2x 

> https://mercury-proxy.privateinternetaccess.com/v3/publisher/identity?publisher=example.com

It can be replaced with a call on https://publishers.basicattentiontoken.org/api/v1/public/channels and a check if the publisher is found within the array "channels"

**Comment by maxenceC** [comment]
It's not a great solution, but it's what I'm am using for now as a replacement

I need to play a bit more with the API to find if there's another way

**Comment by da2x** [comment]
I believe they wanted to do the checking client-side to not leak any information about which domains people were visiting. On the other hand, you now have a complete list of publishers in one go.

**Comment by maxenceC** [comment]
> On the other hand, you now have a complete list of publishers in one go.

Yeah this is great, I don't have to scrap the top 1M on Alexa and verify if the website is a publisher or not, as I already have a full list now

I'm now doing the opposite, gathering the Alexa rank for an already known Brave publisher to display it on the site
